src/en/mobile/android/development/vocal-navigation.md

```diff
@@ -0,0 +1,463 @@
+
+A simple vocalization may be sufficient to notify the user. It is very easy to trigger vocalizations with TalkBack : you just need to use the `announceForAccessibility` method. 
+
+**Note**: the `announceForAccessibility` method is available on any element that inherits from `View`.
+
+It is also possible to specify to a view that it is a live region, i.e. its content can be dynamically modified. This will result in generating vocalizations with TalkBack for example. A typical example of use: on a form, if the user makes an error and an error message appears, the view containing the message must be defined as a live region.  
+
+The method `setAccessibilityLiveRegion` takes as parameter a mode. There are 3 modes:
+
+- `ACCESSIBILITY_LIVE_REGION_NONE`&nbsp;: this view is not a live region. This is the default value for most views.
+- `ACCESSIBILITY_LIVE_REGION_POLITE`&nbsp;: vocalizations are performed on the view changes. These vocalizations are "polite" and have a lower priority than system vocalizations for example.
+- `ACCESSIBILITY_LIVE_REGION_ASSERTIVE`&nbsp;: vocalizations are performed on the view changes. These vocalizations are "&nbsp;rude&nbsp;". They are prioritized and immediately vocalized. 
+
+**Note**: the `setAccessibilityLiveRegion` method is available on any element that inherits from `View`.
+
+
+**To be verified&nbsp;:**
+- Temporary messages displayed on the screen (such as Toast) are vocalized.
+- Dialogs opening are vocalized
+- Transitions between activities are vocalized (by the title of the new activity for example).
+- Error messages are vocalized
+- Horizontal scrolls are vocalized
+- The change of a part of the screen is vocalized.
+- The change of a screen element is vocalized if it is useful for the user to be informed about it.
+
+
+**Example of triggering a vocalization**:
+
+It is common to test the Android version before triggering TalkBack vocalization. Indeed, `announceForAccessibility` only works since Android Jelly Bean version.
+<pre>
+<code class="java">if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+    myView.announceForAccessibility(getString(R.string.criteria_contentchange_ex1_announce));
+    }
+</code></pre><pre>
+<code class="kotlin">if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+    myView.announceForAccessibility(getString(R.string.criteria_contentchange_ex1_announce))
+    }
+</code></pre>
+
+**Example of a live region**:
+
+<pre><code class="xml">&lt;TextView
+   […]
+   android:accessibilityLiveRegion="polite" /&gt;
+</code></pre>
+
+
+## Do not vocalize the decorative and hidden elements.
+
+**Target&nbsp;:** everyone and in particular the visually impaired.  
+**When&nbsp;:** from design,  and during development.
+
+**Description&nbsp;:**  
+
+To not confuse navigation with a screen reader, it is necessary to vocalize only the significant elements. Decorative elements, such as illustrations, should not be vocalized.
```

Comment:
s/To not confuse navigation with a screen reader/To avoid confusion for a screen-reader user/